Makd.mak

```diff
@@ -386,8 +386,8 @@ endef
 RDMDFLAGS ?= --force --compiler=$(DC)
```

Comment:
Cool :+1: Do you care to put that in the commit message maybe (to require imports to do this).